Documentation/build_process.md

```diff
@@ -501,6 +501,67 @@ when packaing Release applications.
 
     Added in Xamarin.Android 7.1.
 
+-   **AndroidVersionCodePattern** &ndash; A string property which allows
+    the developer to customize the `versionCode` in the manifest.
+    See [Creating the Version Code for the APK](https://developer.xamarin.com/guides/android/advanced_topics/build-abi-specific-apks/#Creating_the_Version_Code_for_the_APK)
+    for information on deciding a `versionCode`.
+    
+    Some examples, if `abi` is `armeabi` and `versionCode` in the manifest
+    is `123`
+
+        {abi}{versionCode}
+
+    will prodice a versionCode of `1123`.
```

Comment:
This should be:

> ...will produce [sic] a versionCode of `11123` when `$(AndroidCreatePackagePerAbi)` is True, otherwise will produce a value of `123`.